Gordon360/Services/HousingService.cs

```diff
@@ -13,6 +13,9 @@
 using Microsoft.EntityFrameworkCore;
 using Microsoft.IdentityModel.Tokens;
 using Gordon360.Static.Names;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Graph;
+using System.Net.NetworkInformation;
```

Comment:
These look like unnecessary imports. Are they actually used for anything in this service?